src/coreclr/src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ else()
         ${END_WHOLE_ARCHIVE}
         mscorrc_debug
         palrt
+        System.Globalization.Native-Static
```

Comment:
Just something to double check - is this binplaced a different directory than the one built from libraries/native? We do not want libraries build to be overwriting products of coreclr build.

Reply:
Yeah, they will be binplaced in a different directory because when building coreclr, the output directory will be set by the coreclr cmakelists which points to coreclr output directory.